src/main/java/com/nerodesk/om/mock/MkDoc.java

```diff
@@ -137,6 +138,17 @@ public void write(final InputStream input) throws IOException {
         Logger.info(this, "%s saved", file);
     }
 
+    @Override
+    public String shortUrl() {
+        String url = "No URL";
+        try {
+            url = this.file().toURI().toURL().toString();
+        } catch (final MalformedURLException ex) {
+            Logger.error(this, "Error happened: %s", ex.getMessage());
```

Comment:
hm.. are you serious? swallowing the exception? see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#exceptions